_projects/de/2024-04-02-ToF-LED.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: project_page
+title: "Tiefenbilder auf der LED-Matrix anzeigen"
+date: 2024-04-02
+author: Paula
+abstract: "Lasse dir die 8x8 Distanzmessungen des Time-of-Flight Sensors auf der LED-Matrix anzeigen."
+thumbnail:  /images/projects/Titelbild_ToF-LED.png
+image0: /images/projects/ToF-LED/0.png
+image1: /images/projects/ToF-LED/1.png
+material:
+    - senseBox MCU-S2
+    - Time-of-Flight Distanzsensor
+    - LED-Matrix
+    - 2x Qwicc-Kabel
+ide: blockly
+version: ["edu-S2"]   
+lang: de
+tags: ["Informatik"]
+difficult: leicht
+---
+
+In diesem Projekt kannst du dir die Distanzmessungen des ToF-Imager auf einer LED-Matrix anzeigen lassen. 
+
+Der ToF-Imager kann wie eine Kamera ein Bild seiner Umgebung aufnehmen, jedoch nicht ein Farbbild sondern ein Tiefenbild an Entfernungen. Maximal kann der ToF-Imager 8x8 Distanzen aufnehmen, des entstehende Bild ist deshalb sehr verpixelt. ToF ist eine Abkürzung für die Messmethode "Time-of-Flight". Das bedeutet, dass der Sensor Lichtstrahlen (speziell Infrarotlicht) aussendet und misst wie lange es dauert, bis diese Lichtstrahlen wieder zurückreflektiert werden. Je weiter ein Objekt oder Hindernis entfernt ist, desto länger dauert es, bis das Licht wieder am Sensor ankommt. Unser ToF-Imager kann Distanzen von bis zu 4 Metern messen. Dabei kommt es jedoch darauf an, wie viel anderes Licht bereits in der Umgebung vorkommt, da dieses die Messung der Distanz stört.
+
+## Aufbauen
+Schließe den Distanzsensor per Qwiic Kabel an eine I2C Schnittstelle der senseBox an. Schließe nun die LED-Matrix an der GPIO Schnittstelle "A" an.
+
+{% include block.html image=page.image0 %}
```

Comment:
I think this should be `{% include image.html ...`

Reply:
thanks, now its centered. Still kinda small I guess, but maybe thats fine